openapi-cli/src/test/java/io/ballerina/openapi/generators/client/EnumGenerationTests.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
```suggestion
 * WSO2 LLC. licenses this file to you under the Apache License,
```

Reply:
Updated in https://github.com/ballerina-platform/openapi-tools/pull/1328/commits/1c1b409182ccd309a4da54dd57c18e65241dd1bc